fork-off/README.md

```diff
@@ -31,7 +31,7 @@ The tool will perform the following actions, in this order:
 2. Dump the state to a json file. You can provide a path via `--snapshot-path`.
 3. Read the state from the snapshot json file. This is because steps 1. and 2. can be omitted by running with `--use-snapshot-file` -- see example below.
 4. Read the chainspec provided via `--initial-spec-path` you should pass here the one generated via `the bootstrap-chain` command, so `--initial-spec-path=chainspec.json` if it is in the same directory.
-5. Replace the genesis state in the chainspec by the one from the snapshot WITH THE EXCEPTION of states of pallets provided via a comma separated list using `--pallets_keep_state`. The default setting is `--pallets_keep_state=Aura,Aleph,Sudo,Staking,Session,Elections` and it's likely you don't want to change it.
+5. Replace the genesis state in the chainspec by the one from the snapshot WITH THE EXCEPTION of states of paths provided via a comma separated list using `--storage_keep_state`. The default setting is `--storage_keep_state=Aura,Aleph,Sudo,Staking,Session,Elections,System.Account` and it's likely you don't want to change it.
```

Comment:
might need changes to GH workflows

Reply:
true, I have changed according to @Marcin-Radecki suggestion (removing this arg)